notebooks/community/model_garden/model_garden_pytorch_open_clip.ipynb

```diff
@@ -0,0 +1,677 @@
+{
```

Comment:
<p>Fine tune model -&gt; Fine tune the model</p><p> </p><p>we -&gt; you</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>